puble.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>publishing_m</title>
+    <link rel="stylesheet" href="puble.css" />
+  </head>
+  <body>
+    <body class="container">
```

Comment:
이 코드는 어떤 뜻인가요...?
body 안에 컨텐츠를 모두 담은 container 를 만들고 싶었던 것이라면 body 태그가 아니라 div 태그를 사용해야 합니다!
```suggestion
    <div class="container">
```